R/correctionsataglance-data.R

```diff
@@ -101,12 +101,14 @@ formatDateRange <- function(startD, endD){
     endSeq <- c(startSeq[-1], endD)
   }
   
-  #don't print Month Year in plot if there isn't enough room inside the rectangle
+#   #don't print Month Year in plot if there isn't enough room inside the rectangle
   dateSeq <- startSeq
   startSpan <- as.numeric(endSeq[1] - startSeq[1])
   endSpan <- as.numeric(tail(endSeq, 1) - tail(startSeq, 1))
-  if(startSpan <= 16){dateSeq[1] <- NA}
-  if(endSpan <= 14){dateSeq[length(dateSeq)] <- NA}
+  startLabelText <- format(startD, "%b %Y")
+  endLabelText <-  format(endD, "%b %Y")
+  if(isTextLong(labelText=startLabelText,dateLim=NULL,startD,endD,totalDays=numdays)) {dateSeq[1] <- NA}
+  if(isTextLong(labelText=endLabelText,dateLim=NULL,startD,endD,totalDays=numdays)) {dateSeq[length(dateSeq)] <- NA}
```

Comment:
@lindsaycarr Here you had suggested I use startD=startSpan and endD=endSpan, but I run into errors on that, and it looks like the isTooLong function at line 350, is looking for the difference between two dates, not the number # of days. Anyway, it's still plotting the end month/year for some reason. Can you look at this with me sometime today?